src/ImageSharp/Formats/WebP/LosslessUtils.cs

```diff
@@ -0,0 +1,656 @@
+
+        /// <summary>
+        /// The goal of the color transform is to de-correlate the R, G and B values of each pixel.
+        /// Color transform keeps the green (G) value as it is, transforms red (R) based on green and transforms blue (B) based on green and then based on red.
+        /// </summary>
+        /// <param name="transform">The transform data.</param>
+        /// <param name="pixelData">The pixel data to apply the inverse transform on.</param>
+        public static void ColorSpaceInverseTransform(Vp8LTransform transform, Span<uint> pixelData)
+        {
+            int width = transform.XSize;
+            int yEnd = transform.YSize;
+            int tileWidth = 1 << transform.Bits;
+            int mask = tileWidth - 1;
+            int safeWidth = width & ~mask;
+            int remainingWidth = width - safeWidth;
+            int tilesPerRow = SubSampleSize(width, transform.Bits);
+            int y = 0;
+            int predRowIdxStart = (y >> transform.Bits) * tilesPerRow;
+            Span<uint> transformData = transform.Data.GetSpan();
+
+            int pixelPos = 0;
+            while (y < yEnd)
+            {
+                int predRowIdx = predRowIdxStart;
+                Vp8LMultipliers m = default(Vp8LMultipliers);
+                int srcSafeEnd = pixelPos + safeWidth;
+                int srcEnd = pixelPos + width;
+                while (pixelPos < srcSafeEnd)
+                {
+                    uint colorCode = transformData[predRowIdx++];
+                    ColorCodeToMultipliers(colorCode, ref m);
+                    TransformColorInverse(m, pixelData, pixelPos, tileWidth);
+                    pixelPos += tileWidth;
+                }
+
+                if (pixelPos < srcEnd)
+                {
+                    uint colorCode = transformData[predRowIdx];
+                    ColorCodeToMultipliers(colorCode, ref m);
+                    TransformColorInverse(m, pixelData, pixelPos, remainingWidth);
+                    pixelPos += remainingWidth;
+                }
+
+                ++y;
+                if ((y & mask) is 0)
+                {
+                    predRowIdxStart += tilesPerRow;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Reverses the color space transform.
+        /// </summary>
+        /// <param name="m">The color transform element.</param>
+        /// <param name="pixelData">The pixel data to apply the inverse transform on.</param>
+        /// <param name="start">The start index of reverse transform.</param>
+        /// <param name="numPixels">The number of pixels to apply the transform.</param>
+        public static void TransformColorInverse(Vp8LMultipliers m, Span<uint> pixelData, int start, int numPixels)
+        {
+            int end = start + numPixels;
+            for (int i = start; i < end; i++)
+            {
+                uint argb = pixelData[i];
+                sbyte green = (sbyte)(argb >> 8);
```

Comment:
this green contains "arg" (alpha, red and green), right?
if that's the case, why is it called green here?